recipes/leftbar/contact_row/contact_row_constants.js

```diff
@@ -0,0 +1,5 @@
+export const DEFAULT_CONSTANTS = null;
```

Comment:
Where we are using this?

Reply:
We are not it just gets created by the generator. Removed the export and the const. Will leave the file there in case any constants need to be added in the future.